src/table/pagination-table/components/head/styles.ts

```diff
@@ -17,11 +17,16 @@ export const StyledHeadCell = styled(TableCell, {
   pointer: "cursor",
   verticalAlign: "bottom",
 
-  "&:focus": {
-    boxShadow: "none",
-  },
-
-  "&&:hover": {
+  "&&:focus": {
+    ...(!isNewHeadCellMenuEnabled && {
+      boxShadow: "0 0 0 2px #177fe6 inset, 0 0 0 3px #fff inset",
+    }),
```

Comment:
It's my mistake!
the boxShadow being none might be behind flag, because when flag is on we want to have that blue border around cell. will fix it asap